definitions/media-roku_app/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: MEDIA
+# Represents all instances of a customer's Roku App monitored with New Relic's Roku agent.
+type: ROKU_APP
+
+synthesis:
+  identifier: appId
```

Comment:
This gets assigned from Roku when an app/channel gets published, so it should be unique.

Reply:
What format has this appId? we have some limits on the length so if we are unsure if this id will respect those we should encode it.

Reply:
The Roku docs aren't great - but from our live data we can tell it's numeric (and not that large). I suspect it's just a counter on the Roku side that they increment for each new app, so I doubt we'll hit any length limits.